templates/DonorProfile/donorProfile.jinja

```diff
@@ -19,7 +19,7 @@
         {{address}}
         <br>
         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{{address2}}
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{{address2}}

```

Comment:
This should be reformatted to not require nbsp's. Maybe put all the info in a table. Not urgent, so I made a separate issue for it: #35